xml/System/Array.xml

```diff
@@ -6454,12 +6458,24 @@ int[,,] TDArray = new int[1,1,1];
         <Parameter Name="length" Type="System.Int32" Index="2" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="array">To be added.</param>
-        <param name="index">To be added.</param>
-        <param name="length">To be added.</param>
-        <summary>To be added.</summary>
+        <typeparam name="T">The type of the elements in <paramref name="array" />.</typeparam>
+        <param name="array">The one-dimensional <see cref="T:System.Array" /> of elements of type <typeparamref name="T" /> to reverse.</param>
```

Comment:
```suggestion
        <param name="array">The one-dimensional array of elements to reverse.</param>
```

Reply:
The type is what makes this method different from the old overloads.

Reply:
Same here, I don't think you need to disambiguate in params.